co2_scd4x.ino

```diff
@@ -1,29 +1,48 @@
-/* Includes display */
+/*
+   ESP32S2 Dev Module
+   Required Arduino libraries:
+   - esp32 waveshare epd
+   - Adafruit DotStar
+   - Sensirion I2C SCD4x: https://github.com/Sensirion/arduino-i2c-scd4x
+   - WiFiManager: https://github.com/tzapu/WiFiManager
+*/
+
+/* deep & light sleep */
+#include <esp_sleep.h>
+
+/* eInk display */
 #include "DEV_Config.h"
 #include "EPD.h"
 #include "GUI_Paint.h"
 #include <stdlib.h>
-#include "init.h"
+#define EINK_DO_REFRESH 40
```

Comment:
720 for every hour or every six hours on battery

Reply:
Werde meinen `Change refreshing` wieder raus schmeißen